main.js

```diff
@@ -94,8 +104,11 @@ const createPyProc = () => {
         pyProc = require('child_process').spawn('python', ["-m", "rotkehlchen", "--zerorpc-port", port]);
     }
 
+    pyProc.on('data', (data) => {
```

Comment:
Why did you add this callback? What does it do?

Reply:
I'm taking this out